### PR TITLE
Upgrade to the latest gradle

### PR DESCRIPTION
**Pull request by davidsun**
Note that I also updated the Spotless plugin to the latest version since the older version doesn't work with the latest gradle.

Also, re-formatted the code.

### CODE REVIEW

**Comment by davidsun** [comment]
Seems like the CI is not run.

**Review by dlemures — APPROVED** [review]
RTM

**Comment by davidsun** [comment]
Mind explaining what is "RTM"?

Updated the PR to make sure `./gradlew clean check && ./gradlew -b toothpick-sample/build.gradle clean check` passes.

**Comment by davidsun** [comment]
Please also merge for me. Don't have write access to the repo. Thanks!

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,7 +1,5 @@
-#Fri Apr 08 18:13:41 PDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.2-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
```
Comment (dlemures) [inline gradle/wrapper/gradle-wrapper.properties:5]: do we need to change it also for toothpick-sample/gradle-wrapper?

**Comment by dlemures** [comment]
Just one small comment about something that i missed

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (davidsun) [reply gradle/wrapper/gradle-wrapper.properties:5]: We do. Sorry that I didn't realize that there are two root projects under the same repo so I didn't update gradle for the sample project. Fixed. You probably need to run CI again.

**Comment by dlemures** [comment]
Merged! Thx @davidsun

**Pull request merged**